spotguide/spotguide.go

```diff
@@ -730,6 +795,19 @@ func cicdRepoConfigPipeline(request *LaunchRequest, repoConfig *cicdRepoConfig)
 	return nil
 }
 
+func downloadGithubFile(githubClient *github.Client, owner, repo, file, tag string) ([]byte, error) {
+	reader, err := githubClient.Repositories.DownloadContents(ctx, owner, repo, file, &github.RepositoryContentGetOptions{
+		Ref: tag,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	defer reader.Close()
+
+	return ioutil.ReadAll(reader)
```

Comment:
Add stack trace to the error returned here

Reply:
Note: `emperror.Wrap`/`errors.Wrap` checks if the passed error is nil and return nil respectively, so there is no need for `if err != nil` in terminal statements like this.